{"uid":"d7f97a464dda53e2","name":"test_closed_connection","fullName":"tests.unit.V1.async_db.test_connection#test_closed_connection","historyId":"70857186b48e8824663a6c3936638623","time":{"start":1733109463232,"stop":1733109463538,"duration":306},"description":"Connection methods are unavailable for closed connection.","descriptionHtml":"<p>Connection methods are unavailable for closed connection.</p>\n","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","statusTrace":"self = <allure_pytest.listener.AllureListener object at 0x7f1658a7f7f0>\nfixturedef = <FixtureDef argname='connection' scope='function' baseid='tests/unit/V1/async_db'>\n\n    @pytest.hookimpl(hookwrapper=True)\n    def pytest_fixture_post_finalizer(self, fixturedef):\n        yield\n        if hasattr(fixturedef, 'cached_result') and self._cache.get(fixturedef):\n            container_uuid = self._cache.pop(fixturedef)\n>           self.allure_logger.stop_group(container_uuid, stop=now())\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_pytest/listener.py:190: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/reporter.py:91: in stop_group\n    plugin_manager.hook.report_container(container=group)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:34: in report_container\n    self._report_item(container)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:25: in _report_item\n    with io.open(self._report_dir / filename, 'w', encoding='utf8') as json_file:\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_io.py:93: in open\n    return fake_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:105: in fake_open\n    return fake_file_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:136: in __call__\n    return self.call(*args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:244: in call\n    file_object = self._init_file_object(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:358: in _init_file_object\n    file_object = self.filesystem.create_file_internally(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:2551: in create_file_internally\n    self.raise_os_error(errno.ENOENT, parent_directory)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pyfakefs.fake_filesystem.FakeFilesystem object at 0x7f1657756340>\nerr_no = 2\nfilename = '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\nwinerror = None\n\n    def raise_os_error(\n        self,\n        err_no: int,\n        filename: Optional[AnyString] = None,\n        winerror: Optional[int] = None,\n    ) -> NoReturn:\n        \"\"\"Raises OSError.\n        The error message is constructed from the given error code and shall\n        start with the error string issued in the real system.\n        Note: this is not true under Windows if winerror is given - in this\n        case a localized message specific to winerror will be shown in the\n        real file system.\n    \n        Args:\n            err_no: A numeric error code from the C variable errno.\n            filename: The name of the affected file, if any.\n            winerror: Windows only - the specific Windows error code.\n        \"\"\"\n        message = os.strerror(err_no) + \" in the fake filesystem\"\n        if winerror is not None and sys.platform == \"win32\" and self.is_windows_fs:\n            raise OSError(err_no, message, filename, winerror)\n>       raise OSError(err_no, message, filename)\nE       FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:489: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"global_fake_fs","time":{"start":1733109461818,"stop":1733109461916,"duration":98},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_cache","time":{"start":1733109461818,"stop":1733109461818,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"global_fake_fs::0","time":{"start":1733109463563,"stop":1733109463563,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"trio(run=<function run at 0x7f16596040d0>)"},{"name":"parentSuite","value":"tests.unit.V1.async_db"},{"name":"suite","value":"test_connection"},{"name":"host","value":"fv-az585-603"},{"name":"thread","value":"2097-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.unit.V1.async_db.test_connection"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":109,"skipped":0,"passed":58,"unknown":0,"total":167},"items":[{"uid":"3f2633d4bb2c8239","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/372//#testresult/3f2633d4bb2c8239","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1733023506211,"stop":1733023506563,"duration":352}},{"uid":"9a89eb217d3cba","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/375//#testresult/9a89eb217d3cba","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1733023322279,"stop":1733023322561,"duration":282}},{"uid":"4cae334f08ca06c5","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/371//#testresult/4cae334f08ca06c5","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732936473423,"stop":1732936473916,"duration":493}},{"uid":"6d03a0a2935c50a1","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/374//#testresult/6d03a0a2935c50a1","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732936303784,"stop":1732936304103,"duration":319}},{"uid":"4a775cc1a7635792","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/370//#testresult/4a775cc1a7635792","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732850246770,"stop":1732850247109,"duration":339}},{"uid":"88dc338a46ebe199","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/373//#testresult/88dc338a46ebe199","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732850092126,"stop":1732850092553,"duration":427}},{"uid":"760b2b0f2abc3b69","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/369//#testresult/760b2b0f2abc3b69","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732763863636,"stop":1732763864018,"duration":382}},{"uid":"f2722a325804fa73","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/372//#testresult/f2722a325804fa73","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732763717650,"stop":1732763717981,"duration":331}},{"uid":"5d2e67ba717b5eae","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/368//#testresult/5d2e67ba717b5eae","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732677467264,"stop":1732677467665,"duration":401}},{"uid":"15f75529dc3f1b25","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/371//#testresult/15f75529dc3f1b25","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732677322538,"stop":1732677322934,"duration":396}},{"uid":"754d45696564a852","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/367//#testresult/754d45696564a852","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732590958512,"stop":1732590958880,"duration":368}},{"uid":"39415d88d7faf3fb","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/370//#testresult/39415d88d7faf3fb","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732590812597,"stop":1732590812903,"duration":306}},{"uid":"ae0403538355c5c0","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/366//#testresult/ae0403538355c5c0","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732504639817,"stop":1732504640296,"duration":479}},{"uid":"2d6ff93360dac39d","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/369//#testresult/2d6ff93360dac39d","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732504491391,"stop":1732504491754,"duration":363}},{"uid":"c489df91c02a6bf1","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/365//#testresult/c489df91c02a6bf1","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732418292279,"stop":1732418292585,"duration":306}},{"uid":"9134069a25950cef","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/368//#testresult/9134069a25950cef","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732418139215,"stop":1732418139568,"duration":353}},{"uid":"21eb957e3849e3ab","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/364//#testresult/21eb957e3849e3ab","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732331573115,"stop":1732331573416,"duration":301}},{"uid":"9d563e05b530251e","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/367//#testresult/9d563e05b530251e","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732331429851,"stop":1732331430216,"duration":365}},{"uid":"2db3ee70efefcc48","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/363//#testresult/2db3ee70efefcc48","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732245285897,"stop":1732245286211,"duration":314}},{"uid":"6e0cc7fe9a2dc992","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/366//#testresult/6e0cc7fe9a2dc992","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1732245157549,"stop":1732245157998,"duration":449}}]},"tags":["trio(run=<function run at 0x7f16596040d0>)"]},"source":"d7f97a464dda53e2.json","parameterValues":[]}